index.html

```diff
@@ -0,0 +1,289 @@
+                            <img src="Time.png" alt="">
+                        </nav>
+                        <nav class="right">
+                            <img src="Cellular.png" alt="Cellular">
+                            <img src="Wifi.png" alt="Wifi">
+                            <img src="Battery.png" alt="battery">
+                        </nav>
+                        <nav>
+                            <div class="toolbar1">
+                                <a href="#">My Logo</a>
+                            </div>
+                            <section class="header-images">
+                                <div id="elipse11">
+                                    <img src="Ellipse11.png" alt="">
+                                </div>
+                                <div id="elipse5">
+                                    <img src="Ellipse5.png" alt="">
+                                </div>
+                                <div id="rectangle7">
+                                    <img src="Rectangle7.png" alt="">
+                                </div>
+                                <div id="elipse3">
+                                    <img src="Ellipse3.png" alt="">
+                                </div>
+                                <div id="rectangle4">
+                                    <img src="Rectangle4.png" alt="">
+                                </div>
+                                <div id="elipse8">
+                                    <img src="Ellipse8.png" alt="">
+                                </div>
+                            </section>
+                            <div class="toolbar2">
+                                <img src="Union.png" alt="Menu">
+                            </div>
+                        </nav> 
+                    
+                    </div>
+                </header>
+                <section id="primary">
+                    <div class="name">
+                        <h1>I'm Kelvin Glad to see you!</h1>
+                    </div>
+                    <div class="text">
+                        <p>I'm a software developer! I can help you build a product,
+                            feature or website. Look through someof my work and experience!
+                            If you like what you see and have a project you need coded,
+                            don't hesitate to contact me.
+                        </p>
+                    </div>
+                    <nav id="lets-connect">
+                        <h5>LET'S CONNECT</h5>
+                    </nav>
+                    <div>
+                        <ul class="icons">
+                            <li><img src="Vector.png" alt=""></li>
+                            <li><img src="Linkedin.png" alt=""></li>
+                            <li><img src="Vector1.png" alt=""></li>
+                            <li><img src="Vector3.png" alt=""></li>
+                        </ul>
```

Comment:
- [ ] Social icons are not working as expected I think the path is wrong (src=" "), kindly fix it.
- [ ] Great job using `<ul>` and `<li>` here 💯, However. I suggest using `<a>` tag before the `<img>`, Because these social icons are considered links 👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177230998-dcb2eb3e-ba36-4af8-a68f-30133bdff40a.PNG" width="100%" />  | <img src="https://user-images.githubusercontent.com/45231276/177230684-986ea7af-f9a2-444a-8d6f-c8d6837292ad.png"/> |

Reply:
Kindly Implement this change as requested by previous code reviewer